src/Core/Services/MetadataProviders/DatabaseObjectConverter.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.DataApiBuilder.Config.DatabasePrimitives;
+
+namespace Azure.DataApiBuilder.Core.Services.MetadataProviders
```

Comment:
@Aniruddh25/ @seantleonard is this the right place to keep the converters or you want me to move to different location ?

Reply:
Converters go in `Azure.DataApiBuilder.Config` project in `Converters` folder

Reply:
Though actually, since these converters don't apply to config, and instead apply to objects defined in the MetadataProviders, perhaps create a folder here called `Converters`

Reply:
Added a folder : "Convertors"  inside Metadataproviders folder and moved all the converters inside it.